src/CommunityToolkit.Maui/Converters/StringToListConverter.shared.cs

```diff
@@ -78,11 +78,15 @@ public override IEnumerable<string> ConvertFrom(string? value, object? parameter
 		switch (parameter)
 		{
 			case string[] separators:
-				Separators = separators;
-				return Split(value, Separators.ToArray());
+				if (separators.Any(string.IsNullOrEmpty))
```

Comment:
@brminnick, we have the same verification in Separators property. Can we avoid using the same validation in 2 different places?

Reply:
We can't assign `Separators` here because it'll overwrite the user's assignment. 

I added this Unit Test to ensure we don't accidentally override the `Separators` property again:

```cs
[Fact]
public void StringToListConverter_EnsureParameterDoesNotOverrideProperty()
{
  var converter = new StringToListConverter { Separators = new[] { ",", " " } };
  
  var parameterResult = converter.ConvertFrom("maui/toolkit tests", new[] { "/", " " }, null);
  Assert.Equal(new[] { "maui", "toolkit", "tests" }, parameterResult);
  
  var propertyResult = converter.ConvertFrom("maui,toolkit tests", null, null);
  Assert.Equal(new[] { "maui", "toolkit", "tests" }, propertyResult);
}
```

Plus, the `ArgumentException` message is slightly different when validating `parameter` in `ConvertFrom` versus validating `value` in `Separators.set`:

**`Separators.set` Validation of `value`**
```cs
throw new ArgumentException("A null or an empty string is not a valid separator", nameof(value));
```

**`ConvertFrom` Validation of `parameter`**
```cs
throw new ArgumentException("A null or an empty string is not a valid separator", nameof(parameter));

Reply:
Thank you!